Plugins/Json/include/Acts/Plugins/Json/IndexedSurfacesJsonConverter.hpp

```diff
@@ -99,42 +56,56 @@ void convert(nlohmann::json& jIndexedSurfaces,
   if (castedDelegate != nullptr) {
     // Get the surface updator
     auto indexedSurfaces = std::get<SubDelegateType>(castedDelegate->updators);
-    jIndexedSurfaces = convertImpl<SubDelegateType>(indexedSurfaces);
+    jIndexedSurfaces = IndexedGridJsonHelper::convertImpl<SubDelegateType>(
+        indexedSurfaces, detray);
+    if (detray) {
+      nlohmann::json jAccLink;
+      jAccLink["type"] =
+          DetrayJsonHelper::accelerationLink(indexedSurfaces.casts);
+      jAccLink["index"] = 0;
```

Comment:
In detray this enumerates the grids of a given type. Otherwise, it is better to set it to `std::size_t` max, so it becomes clear that the index of the grid is unknown

Reply:
Done.